README.md

```diff
@@ -9,10 +9,10 @@
 
 Build | Status
 -- | --
-Test coverage | [![codecov](https://codecov.io/gh/gazebosim/gz-utils/branch/main/graph/badge.svg)](https://codecov.io/gh/gazebosim/gz-utils)
-Ubuntu Noble  | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz_utils-ci-main-noble-amd64)](https://build.osrfoundation.org/job/gz_utils-ci-main-noble-amd64)
-Homebrew      | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz_utils-ci-main-homebrew-amd64)](https://build.osrfoundation.org/job/gz_utils-ci-main-homebrew-amd64)
-Windows       | [![Build Status](https://build.osrfoundation.org/job/gz_utils-main-win/badge/icon)](https://build.osrfoundation.org/job/ign_utils-ci-win/)
+Test coverage | [![codecov](https://codecov.io/gh/gazebosim/gz-utils/branch/gz-utils3/graph/badge.svg)](https://codecov.io/gh/gazebosim/gz-utils)
```

Comment:
```suggestion
Test coverage | [![codecov](https://codecov.io/gh/gazebosim/gz-utils/branch/gz-utils3/graph/badge.svg)](https://codecov.io/gh/gazebosim/gz-utils/branch/gz-utils3)
```

Reply:
Fixed in https://github.com/gazebosim/gz-utils/pull/138/commits/8a26d57c8fe9362158f157ece64696013baa7ca2. I used https://app.codecov.io/gh/gazebosim/gz-utils/tree/gz-utils3 since the link with `branch` just redirects to https://app.codecov.io/gh/gazebosim/gz-utils.